src/walletSdk/KeyManager/plugins/ScryptEncrypter.ts

```diff
@@ -0,0 +1,47 @@
+import { decrypt, encrypt } from "../helpers/scryptEncryption";
+import {
+  DecryptParams,
+  EncryptedKey,
+  Encrypter,
+  EncryptParams,
+} from "../../Types";
+
+const NAME = "ScryptEncrypter";
+
+/**
+ * Encrypt keys with scrypt, as they are on StellarX.com.
+ */
+export const ScryptEncrypter: Encrypter = {
+  name: NAME,
+  async encryptKey(params: EncryptParams): Promise<EncryptedKey> {
+    const { key, password } = params;
+    const { privateKey, path, extra, publicKey, type, ...props } = key;
+
+    const { encryptedPhrase, salt } = await encrypt({
+      password,
+      phrase: JSON.stringify({ privateKey, path, extra, publicKey, type }),
+    });
+
+    return {
+      ...props,
+      encryptedBlob: encryptedPhrase,
+      encrypterName: NAME,
+      salt,
+    };
+  },
+
+  async decryptKey(params: DecryptParams) {
+    const { encryptedKey, password } = params;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { encrypterName, salt, encryptedBlob, ...props } = encryptedKey;
```

Comment:
@CassioMG same thing here

Reply:
Same answer from [here](https://github.com/stellar/typescript-wallet-sdk/pull/98#discussion_r1489626194).

We can also infer it through `ScryptEncrypter's test` [here](https://github.com/stellar/js-stellar-wallets/blob/master/src/plugins/ScryptEncrypter.test.ts#L32)